extensions/binding/org.eclipse.smarthome.binding.yahooweather/src/main/java/org/eclipse/smarthome/binding/yahooweather/internal/YahooWeatherHandlerFactory.java

```diff
@@ -21,13 +21,16 @@
 import org.eclipse.smarthome.core.thing.ThingTypeUID;
 import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
 import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandlerFactory;
+import org.osgi.service.component.annotations.Component;
 
 /**
  * The {@link YahooWeatherHandlerFactory} is responsible for creating things and thing
  * handlers.
  *
  * @author Kai Kreuzer - Initial contribution
  */
+@Component(service = ThingHandlerFactory.class, immediate = true, configurationPid = "binding.yahooweather")
```

Comment:
Do you think something will break because of this change? I added the PID so it follows the convention that is also used with the bindings generated with the skeleton.

[__bindingIdCamelCase__HandlerFactory.java](https://github.com/eclipse/smarthome/blob/95830210a7f215823712f0fbca053e938e878950/tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/internal/__bindingIdCamelCase__HandlerFactory.java#L43):
```java
@Component(service = ThingHandlerFactory.class, immediate = true, configurationPid = "binding.${bindingId}")
```